packages/hub/src/lib/oauth-handle-redirect.ts

```diff
@@ -1,28 +1,100 @@
 import { HUB_URL } from "../consts";
 import { createApiError } from "../error";
 
+export interface UserInfo {
+	/**
+	 * OpenID Connect field. Unique identifier for the user, even in case of rename.
+	 */
+	sub: string;
+	/**
+	 * OpenID Connect field. The user's full name.
+	 */
+	name: string;
```

Comment:
except `sub`, none of the user-related fields are necessarily provided, no? (particularly given that it seems you can pass a custom list of scopes to `oauthLoginUrl`)

Reply:
the profile ones are, technically we could not send them but we always do

```ts
	res.json({
		...getIdTokenInfo(auth.u, [...(auth.oauth?.scope ?? []), "profile"]),
```